NitroxPatcher/Patches/Dynamic/BreakableResource_SpawnResourceFromPrefab_Patch.cs

```diff
@@ -0,0 +1,59 @@
+using HarmonyLib;
+using Mono.Cecil.Cil;
+using NitroxClient.GameLogic;
+using NitroxClient.MonoBehaviours;
+using NitroxModel.DataStructures;
+using NitroxModel.Helper;
+using NitroxPatcher.PatternMatching;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using UnityEngine;
+using static System.Reflection.Emit.OpCodes;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+/// <summary>
+/// Synchronizes entities that can be broken and that will drop material, such as limestones...
+/// </summary>
+public class BreakableResource_SpawnResourceFromPrefab_Patch : NitroxPatch, IDynamicPatch
+{
+    public static readonly MethodInfo TARGET_METHOD_ORIGINAL = Reflect.Method(() => BreakableResource.SpawnResourceFromPrefab(default, default, default));
+    public static readonly MethodInfo TARGET_METHOD = AccessTools.EnumeratorMoveNext(TARGET_METHOD_ORIGINAL);
+
+    private static readonly InstructionsPattern SpawnResFromPrefPattern = new()
```

Comment:
Pattern not working, make sure to not confuse between the "Call" and "Callvirt" opcodes, which are different

Reply:
When using `Reflect.*` APIs it should now match any call-like opcode.